src/gdb_server/src/lib.rs

```diff
@@ -0,0 +1,90 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::net::{TcpListener, TcpStream};
+
+pub use arch;
+#[cfg(target_arch = "x86_64")]
+pub use arch::x86_64::regs::setup_sregs;
+#[cfg(target_arch = "x86_64")]
+pub use kernel::loader::elf::{Elf64_Phdr, PT_LOAD};
+pub use kvm_bindings;
+pub use kvm_ioctls::VcpuFd;
+pub use std::sync::mpsc::{Receiver, Sender};
+pub use vm_memory::{
+    ByteValued, Bytes, GuestAddress, GuestMemory, GuestMemoryMmap, GuestMemoryRegion,
+};
+
+pub use gdbstub::GdbStubError;
+#[allow(unused_imports)]
```

Comment:
Can we remove `#[allow(unused_imports)]`? 

The purpose of imports is to have a clear understanding of what is used from other crates, but here we're just importing everything.